src-db/database/sourcedata/AD_ELEMENT.xml

```diff
@@ -24,6 +24,20 @@
 <!--0D7EF88E550C4379B886046521156D5C-->  <ISGLOSSARY><![CDATA[N]]></ISGLOSSARY>
 <!--0D7EF88E550C4379B886046521156D5C--></AD_ELEMENT>
 
+<!--1926046ADA624DC3A0583378CF753147--><AD_ELEMENT>
+<!--1926046ADA624DC3A0583378CF753147-->  <AD_ELEMENT_ID><![CDATA[1926046ADA624DC3A0583378CF753147]]></AD_ELEMENT_ID>
+<!--1926046ADA624DC3A0583378CF753147-->  <AD_CLIENT_ID><![CDATA[0]]></AD_CLIENT_ID>
+<!--1926046ADA624DC3A0583378CF753147-->  <AD_ORG_ID><![CDATA[0]]></AD_ORG_ID>
+<!--1926046ADA624DC3A0583378CF753147-->  <ISACTIVE><![CDATA[Y]]></ISACTIVE>
+<!--1926046ADA624DC3A0583378CF753147-->  <COLUMNNAME><![CDATA[Bundle]]></COLUMNNAME>
+<!--1926046ADA624DC3A0583378CF753147-->  <NAME><![CDATA[Is Bundle]]></NAME>
+<!--1926046ADA624DC3A0583378CF753147-->  <PRINTNAME><![CDATA[Is Bundle]]></PRINTNAME>
+<!--1926046ADA624DC3A0583378CF753147-->  <DESCRIPTION><![CDATA[This field identified the modules that are bundles.]]></DESCRIPTION>
+<!--1926046ADA624DC3A0583378CF753147-->  <HELP><![CDATA[If this record has Is Bundle check marked then this module is a bundle.]]></HELP>
```

Comment:
If this record has **the 'Is Bundle'** check marked, then this module is a bundle